pkg/testdata/test-cluster-identity.json

```diff
@@ -0,0 +1,9 @@
+{
+  "clustername": "cluster1",
+  "assetMetadata": [
```

Comment:
I know we decided with `assetMetadata` in the design, but rethinking/reevaluating. May be `assetList` is better property name.

Reply:
updated to use assetList